src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -64,11 +64,27 @@ internal static TimeSeriesOperation Parse(BlittableJsonReaderObject input)
             return result;
         }
 
+        private const string TimeSeriesFormat = "TimeFormat";
+        private const string UnixTimeFormatMs = "UnixTimeFormatMs";
+        private const string UnixTimeFormatNs = "UnixTimeFormatNs";
+        private static readonly long _epochTicks = new DateTime(1970, 1, 1).Ticks;
```

Comment:
Just to be sure, this shouldn't be `new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).Ticks` ?

Reply:
Let's be on the safe side, done.